updating/update-using-custom-machine-config-pools.adoc

```diff
@@ -0,0 +1,76 @@
+[id="update-using-custom-machine-config-pools"]
+= Performing an in-service software update 
+include::modules/common-attributes.adoc[]
+:context: update-using-custom-machine-config-pools
+
+toc::[]
+
+
+There might be some scenarios where you want to perform a controlled rollout of an update to specific worker nodes to ensure that mission-critical applications stay available during the update process. After the cluster is updated, you can update those nodes at more appropriate time. This is also useful in scenarios where you want to use more than one maintenance window to complete the update process. Delaying the updates in this manner is referred to as *in-service software updates (IPSU)* or *canary updates*.
+
+Using an IPSU can also help with large clusters with hundreds of nodes that cannot be fully updated in your organization's defined maintenance window, or if your organization allows multiple maintenance windows. The IPSU process is *not* a typical update workflow. With larger clusters, it can be a time-consuming process that requires you execute multiple commands. This complexity can result in errors that can affect the entire cluster. It is recommended that you carefully consider whether your organization want to use an IPSU and carefully plan implementing the process before you start.
+
+[NOTE]
+====
+It is not recommended to update the MCPs to different {product-title} versions. For example, do not update one MCP from 4.y.10 to 4.y+1.10 and another to 4.y+1.20.
+This scenario has not beeb tested and might result in an undefined cluster state.
+====
+
+The IPSU process described in this topic involves: 
+
+* creating one or more custom machine config pools (MCPs) 
+* labeling each node that you do not want to  upgrade immediately to move those nodes to the custom MCPs 
+* pausing those custom MCPs, which prevents updates to those nodes
+* performing the cluster update
+* removing the custom labels to move some nodes back to their original MCP which triggers the update on those nodes
+* testing the applications on those nodes to make sure the applications work as expected on those newly-updated nodes
+* removing the custom labels from the remaining nodes in small batches and testing the applications on those nodes 
```

Comment:
Un-pausing the pools is not equal to removing the labels. We should just mention that we need to un-pause the pool in a gradual manner and test the applicaton to make sure it works as expected when it is running on the updated nodes.

Reply:
Fixed this error.